docs/README.md

```diff
@@ -113,10 +118,14 @@ After completing installation of the app...
 
 # Challenges & Wins
 ## 🚧 Challenges
-
+- New technology combined with new vocabulary and concepts 
+- Work-life balance
+- Minimizing scope creep 
 
 ## 🌟 Wins
-
+- Learning how to talk to a back end team to reach a common goal
+- Working with content that interests us
+- Further practice with consulting documentation
 
```

Comment:
Could add something about:
- excellent team communication and collaboration
- actually getting graphql to work properly in the app and stubbed in tests